Assets/HoloToolkit/Input/Scripts/InputSources/InputSourceEventArgs.cs

```diff
@@ -1,94 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-using System;
-using UnityEngine;
-
-namespace HoloToolkit.Unity.InputModule
-{
-    /// <summary>
-    /// Basic event args for an input source event.
-    /// </summary>
-    public class InputSourceEventArgs : EventArgs
```

Comment:
I left the single-instance-event-data feature as it was although I was aware of this issue. I understand that the purpose is to avoid allocations and garbage collections but doesn't these fall into the short lived instances and the GC will reuse memory?